licences-specs/src/test/groovy/uk/gov/justice/digital/hmpps/licences/specs/assessment/CaRiskManagementSpec.groovy

```diff
@@ -0,0 +1,44 @@
+package uk.gov.justice.digital.hmpps.licences.specs.assessment
```

Comment:
"assessment" is the RO phase. Should move this to the "finalchecks" dir and change name to just RiskManagementSpec